c/zss.c

```diff
@@ -667,8 +676,9 @@ static WebPluginListElt* readWebPluginDefinitions(HttpServer *server, ShortLived
               if (identifier && pluginLocation) {
                 JsonObject *pluginDefinition = readPluginDefinition(slh, identifier, pluginLocation, relativeTo);
                 if (pluginDefinition) {
-                  WebPlugin *plugin = makeWebPlugin(pluginLocation, pluginDefinition, internalAPIMap,
-                                                    &idMultiplier, pluginLogLevel);
+                  Storage *storage = makeMemoryStorage(NULL);
```

Comment:
This must be conditional.
makeMemoryStorage is for when apiml does not exist. When apiml does exist, need to use makeApimlStorage.

Reply:
@1000TurquoisePogs I am uncertain about authentication. What I noticed working with Cashing Service is APIML considers the client of the Cashing Service as a service(not a regular user). Most likely, in case of Caching Service we need Apimltoken which is per service(not per user).